docs/core/versions/selection.md

```diff
@@ -141,11 +141,11 @@ The roll-forward behavior for an application can be configured in four different
 
 ### Precedence
 
-Roll forward behavior is considered in the following order:
+Roll forward behavior is considered in the following order when your app is run:
 
-01. `*.runtimeconfig.json` config file.
-01. The `DOTNET_ROLL_FORWARD` environment variable.
-01. Any `--roll-forward` parameter passed to the running application.
+01. First the `*.runtimeconfig.json` config file is evaluated.
```

Comment:
So the precedence is actually the opposite of this order. I think this might be confusing because you might think that item 1 takes highest precedence.